packages/cli/src/__tests__/checkpoint.test.ts

```diff
@@ -0,0 +1,35 @@
+import { jestConsoleContext, jestContext } from '@prisma/sdk'
+
+import { redactCommandArray, SENSITIVE_CLI_OPTIONS, tryToReadDataFromSchema } from '../utils/checkpoint'
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
```

Comment:
You don't really need `jestConsoleContext`, right?
```suggestion
import { jestConsoleContext, jestContext } from '@prisma/sdk'

import { redactCommandArray, SENSITIVE_CLI_OPTIONS, tryToReadDataFromSchema } from '../utils/checkpoint'

const ctx = jestContext.new().add(jestConsoleContext()).assemble()
```

```ts
import { jestContext } from '@prisma/sdk'

import { redactCommandArray, SENSITIVE_CLI_OPTIONS, tryToReadDataFromSchema } from '../utils/checkpoint'

const ctx = jestContext.new().assemble()
```

Reply:
Weird, it shows as a suggestion I cannot just add/commit